src/Algebra/Graph/Relation/InternalDerived.hs

```diff
@@ -45,16 +48,6 @@ instance Ord a => Eq (ReflexiveRelation a) where
 instance (Ord a, Show a) => Show (ReflexiveRelation a) where
     show = show . reflexiveClosure . fromReflexive
 
-instance Ord a => Graph (ReflexiveRelation a) where
-    type Vertex (ReflexiveRelation a) = a
-    empty       = ReflexiveRelation empty
-    vertex      = ReflexiveRelation . vertex
-    overlay x y = ReflexiveRelation $ fromReflexive x `overlay` fromReflexive y
-    connect x y = ReflexiveRelation $ fromReflexive x `connect` fromReflexive y
-
-instance Ord a => Reflexive (ReflexiveRelation a)
-
--- TODO: Optimise the implementation by caching the results of symmetric closure.
 {-|  The 'SymmetricRelation' data type represents a /symmetric binary relation/
 over a set of elements. Symmetric relations satisfy all laws of the
 'Undirected' type class and, in particular, the
```

Comment:
It would be more precises to say "in addition" instead of "in particular".

Reply:
Agree